### PR TITLE
Like functionality

### PR DESCRIPTION
**Pull request by JessicaBarclay**
- User can like multiple posts, but only once per post. Feature does not update in real time, this is a separate feature(see Trello board for more info on WebSockets with Action Cable).

- Functionality utilises [Acts as Votable gem](https://github.com/ryanto/acts_as_votable) which will 'Allow any model to be voted on, like/dislike, upvote/downvote, etc.'

- You will notice in the views that the 'Like' function is rendered using partials, in a separate form view.

Would be great to get this merged, particularly if anyone wants build on this(such as liking photos for example).

nb. You may may need to run `rails db:migrate` when you pull down to create the voting db table.

Thank you in advance, 
Cristina and Jess

### CODE REVIEW

**Review by FiddlersCode — CHANGES_REQUESTED** [review]
image
Wow, cool code, guys!
I assume this isn't meant to happen? :) . Is the number of likes supposed to increase after the like button has been clicked?  It doesn't for me, even after refresh.

**Comment by enonnai** [comment]
Thanks for your feedback @pmuldoon86 . Yes, the number of likes is supposed to increase after a refresh only, as the real time update is a separate feature (there is a ticket about it in 'need research' column). Did you run `rake:db migrate` after pulling down pls?

**Review by danielemanuel — APPROVED** [review]
screen shot 2017-06-01 at 20 44 33

User can like multiple posts, number of likes increases after page refresh, code looks DRY.
To be able to like a post you have to sign in as a user.

**Comment by danielemanuel** [comment]
@pmuldoon86 , if you want to like a post you have to be sign in as a user.

**Comment by sophieklm** [comment]
:shipit: 
nice simple implementation, realtime updates would be good, but a separate more complicated issue

File: spec/web_helpers.rb [hunk 1]
```diff
@@ -9,6 +9,15 @@ def sign_up
   click_button 'Sign up'
 end
 
+def sign_up_2
```
Comment (dbugsy) [inline spec/web_helpers.rb]: Looks like duplication. You could make the original sign_up method accept an argument for email and/or password.

**Review by dbugsy — APPROVED** [review]
Looks awesome consider small refactor in tests.

**Comment by JessicaBarclay** [comment]
@pmuldoon86 seems to be working for others - perhaps a db migration issue locally for you? Will take a look with you in the morning if you wish, just throw me a Zoom 👍 

@sophieklm @danielemanuel cheers - real time updates would be great, but as you said Sophie seems quite complex. Thanks guys

@ddosque Refactored the web_helpers, thank you for the feedback

**Comment by enonnai** [comment]
Thank you @danielemanuel ,@sophieklm and @ddosque! All good to go now.

**Comment by JessicaBarclay** [comment]
@pmuldoon86 `bundle install` then `rails db:migrate` ?

**Review by FiddlersCode — APPROVED** [review]

**Comment by FiddlersCode** [comment]
Nice work, guys!  It ran for me after `bin/rails db:migrate RAILS_ENV=development`

**Comment by enonnai** [comment]
thank you @pmuldoon86 ! But you didn't give us an inspector squirrel haha, just joking ;)

**Pull request merged**